server/testutils/factories/person.ts

```diff
@@ -8,6 +8,7 @@ export default Factory.define<Person>(({ params }) => {
   const currentPrison = params.currentPrison || `${county} (HMP)`
 
   return {
+    bookingId: faker.string.numeric({ length: 10 }),
```

Comment:
I'm not sure if these are sequential, but the example booking IDs appear to have a length of 6, might be worth updating here, and in the `prisoner` factory. They also come back as an integer so it's probably worth matching that on our own models too.

Reply:
My mistake, this is a string (it's an integer in the Prison API for some reason)